cpp/order/Nematic.cc

```diff
@@ -55,9 +47,8 @@ void Nematic::compute(quat<float>* orientations, unsigned int n)
     util::forLoopWrapper(0, n, [&](size_t begin, size_t end) {
         for (size_t i = begin; i < end; ++i)
         {
-            // get the director of the particle
-            quat<float> q = orientations[i];
-            vec3<float> u_i = rotate(q, m_u);
+            // get the orientation of the particle and normalize it
+            vec3<float> u_i = orientations[i] / std::sqrt(dot(orientations[i], orientations[i]));
```

Comment:
```suggestion
            // get the orientation of the particle and normalize it
            auto u_i = orientations[i];
            u_i = u_i / std::sqrt(dot(u_i, u_i));
```
A compiler may be smart enough to optimize what was previously written to what I've suggested, but we might as well slightly change the way the code is written to give the compiler a little help.